### PR TITLE
2.20.0 release blog post

### PR DESCRIPTION
**Pull request by cburroughs**
ref https://github.com/pantsbuild/pants/issues/20578

### CODE REVIEW

**Comment by cburroughs** [comment]
Took a stab this morning, long on festive emojis, short on prose.

image

**Review by benjyw — APPROVED** [review]

**Comment by benjyw** [comment]
Thanks for writing this up!

File: blog/2024-03-27-pants-2-20/index.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+_Highlights in 2.20 include_:
+
+- ⚡ Support for the Ruff formatter, for both `BUILD` files and normal Python files.
+- 🔒 Built-in support for Terraform lockfiles.
+- 📁 Support for more remote-caching providers: file system and GitHub Actions Cache.
+- 🪥 New helpers for defining "adhoc" code-quality tools, without requiring a full plugin.
+- ☕ JVM third-party artifacts can now be read from `pom.xml` files, and other related improvements.
+- 🐿 Go 1.22 can now be used.
+- 🥚 Several featuers of the [Pex](https://docs.pex-tool.org/) tool are now exposed by Pants including [`sh_boot`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#sh_boot) [`executable`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#executable), and [`check`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#check).
```
Comment (huonw) [inline blog/2024-03-27-pants-2-20/index.mdx:25]:
```suggestion
- 🥚 Several features of the [Pex](https://docs.pex-tool.org/) tool are now exposed by Pants including [`sh_boot`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#sh_boot) [`executable`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#executable), and [`check`](https://www.pantsbuild.org/2.20/reference/targets/pex_binary#check).
```

File: blog/2024-03-27-pants-2-20/index.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+We are pleased to announce Pants 2.20.0, the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.20.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/docs/getting-started).
+
+_Highlights in 2.20 include_:
+
+- ⚡ Support for the Ruff formatter, for both `BUILD` files and normal Python files.
+- 🔒 Built-in support for Terraform lockfiles.
+- 📁 Support for more remote-caching providers: file system and GitHub Actions Cache.
+- 🪥 New helpers for defining "adhoc" code-quality tools, without requiring a full plugin.
+- ☕ JVM third-party artifacts can now be read from `pom.xml` files, and other related improvements.
```
Comment (huonw) [inline blog/2024-03-27-pants-2-20/index.mdx:23]: What do you think about adding some more links here, similar to the later points?

```suggestion
- ⚡ Support for the Ruff formatter, for both `BUILD` files and normal Python files.
- 🔒 Built-in support for [Terraform lockfiles](https://www.pantsbuild.org/2.20/docs/terraform#lockfiles).
- 📁 Support for [more remote-caching providers](https://www.pantsbuild.org/2.20/docs/using-pants/remote-caching-and-execution/remote-caching): file system and GitHub Actions Cache.
- 🪥 New helpers for [defining "adhoc" code-quality tools](https://www.pantsbuild.org/2.20/reference/targets/code_quality_tool), without requiring a full plugin.
- ☕ JVM third-party artifacts [can now be read from `pom.xml` files](https://www.pantsbuild.org/2.20/reference/targets/jvm_artifacts), and other related improvements.
```

**Review by huonw — COMMENTED** [review]
Looks great, thank you!

I don't know if we have a convention for this yet, but could you also add the corresponding row to `src/pages/versions/index.md` so that it's in place as soon as this is merged?

**Comment by cburroughs** [comment]
> I don't know if we have a convention for this yet, but could you also add the corresponding row to src/pages/versions/index.md so that it's in place as soon as this is merged?

Done.

**Review by huonw — APPROVED** [review]

**Pull request merged**